.github/workflows/test_and_deploy.yml

```diff
@@ -109,9 +123,17 @@ jobs:
 
       - uses: actions/download-artifact@v2
         with:
-          name: wasm bin
+          name: wasm public
+          path: public
+
+<<<<<<< HEAD
```

Comment:
Is that leftover of some merge ?

Reply:
Yeah, those are some weird commits from way behind - must have been on the local branch and i didn't notice them. I hope nothing else is broken